docs/source/WritingTests/WritingSimpleTests.rst

```diff
@@ -167,6 +162,45 @@ to pick up a living guest, connect to it via ssh, and return its uptime.
            logging.info("Guest uptime result is: %s", uptime)
            session.close()
 
+#. Note that all failures that might happen here are implicitly handled
+   by the methods called. If a test went from its beginning to its end
+   without unhandled exceptions, avocado assumes the test automatically
+   as **PASS**, *no need to mark a test as explicitly passed*. If you
+   have explicit points of failure, for more complex tests, you might
+   want to mark it explicitly. ``test.cancel`` makes the test **CANCEL**
+   (for avocado 36lts version, as this method not backport yet, you may
```

Comment:
It is a new feature, so it will never be backported, so to not give false hopes I'd say `(it is a new feature not supported in avocado 36lts ...`.